arrow_pd_parser/_writers.py

```diff
@@ -92,28 +147,25 @@ def write(
             warning_msg = (
                 f"Your kwargs for index ({kwargs_index}) mismatches the writer's "
                 f"settings self.drop_index ({self.drop_index}). "
-                "In this instance kwargs superseeds the writer settings."
+                "In this instance kwargs supersedes the writer settings."
             )
             warnings.warn(warning_msg)
         else:
             kwargs["index"] = not self.drop_index
 
-        if is_s3_filepath(output_path):
-            wr.s3.to_csv(df_out, output_path, **kwargs)
-        else:
-            os.makedirs(os.path.dirname(output_path), exist_ok=True)
-            df_out.to_csv(output_path, **kwargs)
+        df_out.to_csv(f, header=first_chunk, **kwargs)
```

Comment:
dropped this because:
- The requirement set out was that data had to be written the same way across the api
- using wrangler would mean the behaviour was different when writing iterable dataframes as it had to have `dataset=True`
- if you want to use wrangler features, probably just use wrangler
Thanks @mratford :)